reference-architecture/vmware-ansible/roles/openshift-auth-proxy/tasks/main.yaml

```diff
@@ -0,0 +1,51 @@
+---
```

Comment:
Its not used at all. It was a remnant from my predecessor. I've removed the role completely. I do all the auth config in the openshift_install playbook.